data/campaigns/Under_the_Burning_Suns/scenarios/04_Descending_into_Darkness.cfg

```diff
@@ -342,13 +342,26 @@
             [/then]
         [/if]
 
-        {CLEAR_VARIABLE random,sneak_up}
+        {CLEAR_VARIABLE random}
     [/event]
 
     # starting dialogue
     [event]
         name=start
 
+        [if]
+            [variable]
+                name=sneak_up
+                boolean_equals=yes
+            [/variable]
+            [then]
+                [message]
+                    speaker=Nym
+                    message= _ "Thanks to that map we got from the orcs last night, our scouts have been able to confirm the layout of the surrounding terrain, including the location of an orcish camp up ahead! I wonder if we could sneak around without alerting them. Oh my, what’s that far off in the distance? Are those..."
```

Comment:
The logic of "map -> scouts -> better knowledge of terrain" doesn't click for me. The map could have shown the scouts some vantage points, but for the scouts to have seen anything without going deeper into the map than the starting point of the next scenario we have accept that the scouts have a much larger vision range in the cutscene than they do on the actual map.

The logic of "map -> better knowledge of terrain" seems simpler, and doesn't promote the players to overthink the scouts' vision range.